src/libraries/SparkLiquidityLayerHelpers.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import { IERC20 }   from 'forge-std/interfaces/IERC20.sol';
+import { IERC4626 } from 'forge-std/interfaces/IERC4626.sol';
+
+import { IAToken } from "aave-v3-origin/src/core/contracts/interfaces/IAToken.sol";
+
+import { IMetaMorpho, Id }       from "metamorpho/interfaces/IMetaMorpho.sol";
+import { IMorpho, MarketParams } from "morpho-blue/src/interfaces/IMorpho.sol";
+import { MarketParamsLib }       from "morpho-blue/src/libraries/MarketParamsLib.sol";
+
+import { RateLimitHelpers, RateLimitData } from "spark-alm-controller/src/RateLimitHelpers.sol";
+
+/**
+ * @notice Helper functions for Spark Liquidity Layer
+ */
+library SparkLiquidityLayerHelpers {
+
+    // This is the same on all chains
+    address private constant MORPHO = 0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb;
+
+    bytes32 private constant LIMIT_4626_DEPOSIT  = keccak256("LIMIT_4626_DEPOSIT");
+    bytes32 private constant LIMIT_4626_WITHDRAW = keccak256("LIMIT_4626_WITHDRAW");
+    bytes32 private constant LIMIT_AAVE_DEPOSIT  = keccak256("LIMIT_AAVE_DEPOSIT");
+    bytes32 private constant LIMIT_AAVE_WITHDRAW = keccak256("LIMIT_AAVE_WITHDRAW");
+
+    /**
+     * @notice Onboard an Aave token
+     * @dev This will set the deposit to the given numbers with
+     *      the withdraw limit set to unlimited.
+     */
+    function onboardAaveToken(
+        address rateLimits,
+        address token,
+        uint256 depositMax,
+        uint256 depositSlope
+    ) internal {
+        IERC20 underlying = IERC20(IAToken(token).UNDERLYING_ASSET_ADDRESS());
+
+        RateLimitHelpers.setRateLimitData(
+            RateLimitHelpers.makeAssetKey(
+                LIMIT_AAVE_DEPOSIT,
+                token
+            ),
+            rateLimits,
+            RateLimitData({
+                maxAmount : depositMax,
+                slope     : depositSlope
+            }),
+            "atokenDepositLimit",
+            underlying.decimals()
+        );
+        RateLimitHelpers.setRateLimitData(
```

Comment:
does it make sense to add the withdraw rate limit as a parameter, for the function call to more closely resemble the forum post's instructions? or do we foresee it to always be unlimited?

Reply:
In every case this should be unlimited. There is no danger here apart from bad UX of the borrower. The code leaves it open for some exception, but it's unclear what that would be at this point. If/when that happens we can always call the underlying directly, but these should be set up for the common case.